python/tskit/trees.py

```diff
@@ -3070,6 +3070,117 @@ def simplify(
         else:
             return new_ts
 
+    def slice(
+            self, start=None, stop=None, reset_coordinates=True, simplify=True,
+            record_provenance=True):
+        """
+        Truncate this tree sequence, returning a tree sequence that spans a reduced
+        genomic interval. As with all tree sequence coordinates, the interval is treated
+        as half closed, that is, the leftmost point is included and rightmost point
+        excluded. Sites not covered by the sliced region are thrown away.
+
+        :param float start: The leftmost genomic position, giving the start point
+            of the kept region. Tree sequence information along the genome prior to (but
+            not including) this point will be discarded. If None, use the start position
+            of the current tree sequence.
```

Comment:
This is always equal to 0, no? Shouldn't it be "If None, is set equal to 0."?

Reply:
Done